app/controllers/auth/registrations_controller.rb

```diff
@@ -115,6 +116,12 @@ def set_invite
     end
   end
 
+  def set_invite_reason_message
+    # rubocop:disable Rails/OutputSafety
+    @invite_reason_message = markdown.render(Setting.require_invite_message).html_safe if Setting.require_invite_message.present?
+    # rubocop:enable Rails/OutputSafety
```

Comment:
I'm not sure how to do this safely? Apparently markdown.render returns html as a string but it's not marked as safe for usage in templates?